hcloud/action.go

```diff
@@ -125,3 +125,62 @@ func (c *ActionClient) All(ctx context.Context) ([]*Action, error) {
 
 	return allActions, nil
 }
+
+// Wait watches the action state until it completes with success or error.
+// Also provides progress updates on the first channel.
+func (c *ActionClient) Wait(ctx context.Context, action *Action) (<-chan int, <-chan error) {
```

Comment:
I don’t think Wait is a good name because it does not wait. Maybe Watch or WatchProgress?